configs/opencv-ovms/cmd_client/main.go

```diff
@@ -29,9 +30,14 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"sync"
 	"time"
 
+	"github.com/intel-retail/vision-self-checkout/configs/opencv-ovms/cmd_client/parser"
 	"github.com/intel-retail/vision-self-checkout/configs/opencv-ovms/cmd_client/portfinder"
+
+	grpc_client "github.com/intel-retail/vision-self-checkout/configs/opencv-ovms/cmd_client/grpc-client"
```

Comment:
should this URL be `github.com/intel-retail/automated-self-checkout` ?

Reply:
it is related to go.mod module definition: i can change all the vision-self-checkout to automated-self-check in the current code including the go.mod.

Reply:
@antoniomtz fixed.